daemon/src/model.rs

```diff
@@ -635,32 +638,131 @@ impl Default for FundingRate {
     }
 }
 
-/// Fee paid on every contract renewal (a fraction of it if rolling over)
-#[derive(Debug, Clone)]
+/// Fee paid by the taker to the maker for opening a cfd.
+#[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Serialize, Deserialize)]
+pub struct OpeningFee {
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
```

Comment:
Is there a particular reason why you opted for serialization `as_btc`?
I think we should have a separate type for the projection. Internally I think we should always serialize `as_sat`.
We only use `as_btc` in the projection at the moment, which should also be changed to `as_sat` (see https://github.com/itchysats/itchysats/issues/1079)

Note that a lot of the structs defined in `model` should actually not live there, but should be moved into `projection`, but that is out of scope for this PR.

Reply:
I opted for this for consistency, `Amount` everywhere is serialized this way - however, as @thomaseizinger pointed out, might need to change to sat anyway.

Reply:
it's just `u64` wrapper now, as per other comments - problem went away :)